README.md

```diff
@@ -1,3 +1,5 @@
+# Important: The integration will reach End of Support on September 30, 2024. After that date, this repository will no longer receive updates from JFrog and will no longer be supported by JFrog
```

Comment:
You may want to use GitHub Markdown's alert syntax to make this more prominent. https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts